src/test/data/JsonSerializablePurchaseHistoryTest/invalidFood.json

```diff
@@ -0,0 +1,27 @@
+{
+  "purchases" : [ {
+    "purchasedFood" : {
+      "name" : "Carbonara",
+      "price" : "5.00",
+      "description" : "Italian noodle with cheese and meat",
+      "category": "Italian",
+      "tagged" : [ "Noodle" ],
+      "location" : "The Deck",
+      "openingHours" : "0800 1900",
+      "restrictions" : "Contains dairy and pork"
+    },
+    "timeOfPurchaseInMillisSinceEpochString" : "abc"
```

Comment:
This should be an InvalidFoodPurchase test isn't it? Since here, you are testing for timeOfPurchase to be some Alphaberts.